coilsnake/ui/cli.py

```diff
@@ -72,14 +89,46 @@ def _upgrade(args):
                     project_path=args.project_directory)
 
 def _scriptdump(args):
-	decompile_script(rom_filename=args.rom_filename, 
-					 project_path=args.project_directory)
+    decompile_script(rom_filename=args.rom_filename, 
+                     project_path=args.project_directory)
 
 def _patchrom(args):
-	patch_rom(clean_rom_filename=args.clean_rom,
-			  patched_rom_filename=args.output_rom,
-			  patch_filename=args.patch,
-			  headered=args.headered)
+    if args.headered == "true":
+        header = True
+    else:
+        header = False
+    patch_rom(clean_rom_filename=args.clean_rom,
+              patched_rom_filename=args.output_rom,
+              patch_filename=args.patch,
+              headered=header)
+
+def _expand(args):
+    if args.exhi == "true":
```

Comment:
i am also confused here. i combined the two expand commands into one like you said, and tested it, and it works exactly as it should. using `exval = (args.exhi == "true")` gives an incorrect syntax error, and `exval == (args.exhi == "true")` says that you're referencing a variable that doesn't exist yet. again, could you tell me what problem you're seeing with the code as is?